scripts/core/ui/components/ListPage/generic-list-page.tsx

```diff
@@ -175,35 +175,35 @@ export class GenericListPageComponent<T extends IBaseRestApiResponse>
             },
         }), callback);
     }
+    validateFilters(filterValues) {
+        return Object.keys(filterValues).reduce((accumulator, key) => {
+            const value = filterValues[key];
+
+            if (typeof value === 'string') {
+                let trimmedValue = value.trim();
+
+                if (trimmedValue.length > 0) {
+                    accumulator[key] = trimmedValue;
+                    return accumulator;
+                } else {
+                    return accumulator;
```

Comment:
it could be like:
```
if (trimmedValue.length > 0) {
    accumulator[key] = trimmedValue;
}

return accumulator;
```

Reply:
I didn't write this function so I didn't really look into the implementation, but yeah you're right that's quite redundant